wiki/Tournaments/OMMT/3/en.md

```diff
@@ -0,0 +1,481 @@
+| **Ju1nY11** ::{ flag=MY }:: | **5** | 0 | ::{ flag=MY }:: GG\_com2016 | [#1](https://osu.ppy.sh/community/matches/109392010) |
+| 256 ::{ flag=MY }:: | 2 | **5** | ::{ flag=MY }:: **kaijunchang7628** | [#1](https://osu.ppy.sh/community/matches/109392486) |
+| **\[RGK\]Cheoky29** ::{ flag=MY }:: | **5** | 0 | ::{ flag=MY }:: xyyyyyy | [#1](https://osu.ppy.sh/community/matches/109392860) |
+
+Sunday, 9 July 2023:
+
+| Player A |  |  | Player B | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **JayLye** ::{ flag=MY }:: | **5** | 0 | ::{ flag=MY }:: PurpleLune | [#1](https://osu.ppy.sh/community/matches/109407572) |
+| **LloydTTG** ::{ flag=MY }:: | **5** | 0 | ::{ flag=MY }:: XxTheJKxX | [#1](https://osu.ppy.sh/community/matches/109408032) |
+| **ReJust** ::{ flag=MY }:: | **5** | 0 | ::{ flag=MY }:: Kzon | [#1](https://osu.ppy.sh/community/matches/109408709) |
+
+Monday, 10 July 2023:
+
+| Player A |  |  | Player B | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Evirir** ::{ flag=MY }:: | **5** | 1 | ::{ flag=MY }:: cirinoes | [#1](https://osu.ppy.sh/community/matches/109420495) |
+| Jugemu-Jugemu ::{ flag=MY }:: | 2 | **5** | ::{ flag=MY }:: **RedCatzzButReal** | [#1](https://osu.ppy.sh/community/matches/109422830) |
+
+### Qualifiers
+
+Detailed statistics for this round can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/11U3G20HprnsCz5e5PUmJCwVtPBbxwGFI3DJkieeDBPE/edit) alongside the final standings on which players made it through to the next stage.
+
+## Ruleset
+
+Every staff and player must respect each other and follow the [osu! community rules](/wiki/Rules), no discussion about sensitive topics (example: religion, politics) and anything that could be offensive to others. This is a friendly 1v1 4k national tournament and not the place for such discussions. If a player breaks any of the osu! community rules, the player will be punished. Tournament staff have the right to punish players that show repeated and excessive toxic behavior toward any staff member, player, or spectator either in match multiplayer chatroom, the official discord server, or the official twitch channel chatroom.
+
+Punishment for violating the osu! community rules include:
+
+- Declaring the match as forfeited, or as a win by default for the other player.
+- Disqualification from the entire tournament.
+
+### Qualifiers procedure
+
+1. The referee will notify the player 10-15 minutes before the lobby is scheduled to start. Once all the players are present or another 10 minutes has passed since the scheduled time, the lobby will start.
+2. Players will play the mappool in order from Stage 1-5 twice. After the first cycle, players will be given a 5 minute break before continuing with the 2nd cycle which follows the same procedure. Players are allowed to skip this break if all players present in the lobby agrees to start the 2nd run right away.
```

Comment:
```suggestion
1. The referee will notify the player 10-15 minutes before the lobby is scheduled to start. Once all the players are present or another 10 minutes have passed since the scheduled time, the lobby will start.
2. Players will play the mappool in order from Stage 1-5 twice. After the first cycle, players will be given a 5 minute break before continuing with the 2nd cycle which follows the same procedure. Players are allowed to skip this break if all players present in the lobby agree to start the 2nd run right away.
```